UploadAndAnnotateImages/force-app/main/default/lwc/imageAnnotate/imageAnnotate.js

```diff
@@ -0,0 +1,254 @@
+import { LightningElement, api } from 'lwc';
+import LightningConfirm from 'lightning/confirm';
+import { log, Shapes } from "c/utilsImageCapture";
+
+export default class ImageAnnotate extends LightningElement {
+    @api
```

Comment:
Nit: this line generates the TS1219 warning in VSCode, regarding experimental decorators. We should add a `jsconfig.json` [configuration](https://code.visualstudio.com/docs/languages/jsconfig) under `lwc/` that sets:

```json
"compilerOptions": {
    "experimentalDecorators": true
}
```

So this warning won't show up in VSCode for any of the LWCs.

Reply:
Thanks @khawkins.
This file exists locally, but it's ignored according to `.gitignore` (and `.forceignore`).
Should I remove it from the ignore files and add it to the repo?

Reply:
Yes please, let's make it a part of the repo. Let's remove the entry from `.gitignore`, though I think it's okay to remain in `.forceignore`, as it's not necessary for that file to end up in your Org source. Just the repo itself (for development).

Reply:
Done:)